routes/web.php

```diff
@@ -53,6 +53,8 @@
 
     // Profile Controller
     Route::get('/profile', 'ProfileController@index');
+    Route::put('/profile', 'ProfileController@update');
+    Route::put('profile/password', 'ProfileController@updatePassword');
```

Comment:
simple styling suggestion
```suggestion
    Route::put('/profile/password', 'ProfileController@updatePassword');
```